src/pages/showcase/_components/ShowcaseCard/index.tsx

```diff
@@ -1,7 +1,7 @@
 
 import React from 'react';
 import clsx from 'clsx';
-import Image from '@theme/IdealImage';
+// import Image from '@theme/IdealImage';
```

Comment:
why use comment

Reply:
because it's use of in our code 

![image](https://github.com/CodeHarborHub/codeharborhub.github.io/assets/99037494/f7cc243a-aaaf-4a33-9dbc-2db0c24cb682)